code/game/machinery/doors/airlock.dm

```diff
@@ -780,7 +780,7 @@ GLOBAL_LIST_EMPTY(airlock_emissive_underlays)
 				visible_message("<span class='warning'>[user] headbutts the airlock.</span>")
 				var/obj/item/organ/external/affecting = H.get_organ("head")
 				H.Weaken(10 SECONDS)
-				if(affecting.receive_damage(10, 0))
+				if(affecting?.receive_damage(10, 0))
```

Comment:
This is potentially unsafe. I don't know when this could happen outside of admin nonsense, but better to be safe. You should istype check affecting.